src/docfx/lib/ProcessUtility.cs

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Docs
+{
+    internal static class ProcessUtility
```

Comment:
>internal [](start = 4, length = 8)

keep consistency... change all Utility classes to `internal` then?

Reply:
process utility will only be used in gitutility, so i mark it as internal

---
In reply to: [180613969](https://github.com/dotnet/docfx/pull/2611#discussion_r180613969) [](ancestors = 180613969)

Reply:
I'd rather Utility classes under lib folder have the same protection level. It is now used only by Git, but it can be used by others

---
In reply to: [180614835](https://github.com/dotnet/docfx/pull/2611#discussion_r180614835) [](ancestors = 180614835,180613969)

Reply:
ok


---
In reply to: [180616419](https://github.com/dotnet/docfx/pull/2611#discussion_r180616419) [](ancestors = 180616419,180614835,180613969)